docs/_data/groups/2017.yml

```diff
@@ -80,6 +80,19 @@
     - name: Eric Poe
       email: eric.poe@gmail.com
 
+- name: Madison PHP
+  location: Madison, WI, USA
+  website: http://www.madisonphpconference.com
+  meetup: https://www.madisonphp.com
```

Comment:
This URL is timing out for me, but when I request it without "https," it properly redirects to your group on Meetup.com. Can you check on the URL and update the pull request, if needed? Thanks!

Reply:
Hi Ben, oh no - I didn’t - thanks for getting back to me! You’re right the Madison PHP should be without the s (so http://www.madisonphp.com , which then redirects to Meetup, which has https on it). I - pushed the change, which seems to have updated the pull request - let me know if you need anything else. Thanks for organizing this!